include/wsman-client-api.h

```diff
@@ -818,11 +818,11 @@ typedef enum {
 
 
 	void
-	wsmc_set_sub_expiry(int event_subscription_expire, client_opt_t * options);
+	wsmc_set_sub_expiry(float event_subscription_expire, client_opt_t * options);
```

Comment:
This effectively changes the API :-/ So we need to increase the `.so` number of the library.